ergoCubSN001/wrappers/inertials/alljoints-inertials_wrapper.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE devices PUBLIC "-//YARP//DTD yarprobotinterface 3.0//EN" "http://www.yarp.it/DTD/yarprobotinterfaceV3.0.dtd">
+
+<device xmlns:xi="http://www.w3.org/2001/XInclude" name="alljoints-inertials_wrapper" type="multipleanalogsensorsserver">
+    <param name="period">      10                           </param>
+    <param name="name"> /${portprefix}/alljoints/inertials   </param>
+
+    <action phase="startup" level="15" type="attach">
+        <paramlist name="networks">
+            <elem name="FirstStrain"> alljoints-inertials_remapper </elem>
+        </paramlist>
+       </action>
+
+    <action phase="shutdown" level="20" type="detach" />
```

Comment:
Kind of related to https://github.com/robotology/robots-configuration/issues/377 . I would use an higher level in detach here w.r.t. to the level used by the remapper to which this is attached, to avoid that the wrapper at some point interacts with a wrapper that has been already detached.

Reply:
Hi @traversaro, could it work with a detach level 25?

Reply:
Yes, but can't we decrease instead the nws to 15? To be compatible with https://github.com/robotology/robots-configuration/issues/377 .